doc/admin-guide/files/records.config.en.rst

```diff
@@ -1347,6 +1347,25 @@ Parent Proxy Configuration
    ``2`` Mark the host down. This is the default.
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.http.parent_proxy.enable_parent_timeout_markdowns INT 0
+   :reloadable:
+   :overridable:
+
+   Enables (``1``) or disables (``0``) parent proxy mark downs due to inactivity
+   timeouts.  The default is disabled (``0``).  This should be used for specific
+   remap entries using ``header_rewrite`` to enable inactivity timeout markdowns rather
```

Comment:
Did you mean conf_remap, not header_rewrite?

Reply:
@ywkaras I mean using the header_rewrite plugin to override the default value of the config so as to enable marking down on timeouts or by disabling markdowns altogether for a particular remap.  Should I change header_rewrite to header_rewrite plugin?

Reply:
So, I thought we normally used this plugin to change config values per remap rule:  https://docs.trafficserver.apache.org/admin-guide/plugins/conf_remap.en.html

Reply:
Oh OK, you can do it with both:  https://docs.trafficserver.apache.org/admin-guide/plugins/header_rewrite.en.html#set-config .  Is there reason to think these would only ever be overridden by header_rewrite?  Maybe it's sufficient that they are marked as overridable?

Reply:
@ywkaras I've updated the documentation.  Please take a look.

Reply:
@ywkaras thanks for the link.  I've never used the conf_remap plugin.  We've always used header_rewrite plugin for that purpose.  I'll change the docs again :)

Reply:
@ywkaras the docs now mention both plugins.